### ISSUE TITLE
face and body?

### ISSUE THREAD

**Issue by Adorablepet** [hqqasw/person-search-PPCC#2]
Thank you very much for your contribution. 
1. After I read the paper, I was a bit confused. What kind of form does the face and body correspond to? I am currently downloading CSM on Baidu's network disk, which is really slow.
2. After I looked at propagation.py, the face and body matrices, did you train them on the CSM, and then took their correspondence out as input to the label propagation algorithm?

**Comment by hqqasw** [comment]
1. The face matrix means the affinity matrix of face, which is constructed by: face detection -> face feature extraction -> affinity matrix calculation by face feature. You can get it from your own pipleline or directly use the affinity matrix we provide. So as the body matrix.
2. Two feature extractors (CNN) is trained on the training set of CSM (one for face and one for body), which are used to extract features on the test set. Then we calculate affinity matrices with the features. And the affinity matrices are the input of LP.

**Comment by Adorablepet** [comment]
@hqqasw Thanks.You don't seem to answer my first question.For example, there are 5 faces and 5 bodies. How does this correspond to face and body?Which face is which body?

**Comment by hqqasw** [comment]
Sorry for misunderstand. We first calculate the IOU of all faces and bodies. And then use bipartite matching to link the faces and bodies.

**Comment by Adorablepet** [comment]
@hqqasw Thanks, I understand.
